### PR TITLE
Adding to cart from item pages #4

### PR DESCRIPTION
**Pull request by concach**
update README and write spec for adding to cart from item show page

### CODE REVIEW

File: spec/features/visitor_adds_items_to_cart_from_items_pages_spec.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      # and I click add to cart
+      click_button("Add to Cart")
+      
+      # I see a message that the item was added to my cart
+      expect(page).to have_content(" has been added to your cart")
+      # and my cart is incremented by 1
+      expect(page).to have_content("Cart: 1")
+    end
+  end
+end
```
Comment (zackforbing) [inline spec/features/visitor_adds_items_to_cart_from_items_pages_spec.rb]: I changed the name of the file to follow convention (unauthenticated user = "visitor")

**Comment by zackforbing** [comment]
looks good, I made a change to the test's filename, please review before merging

File: README.md [hunk 2]
```diff
@@ -1,41 +1,21 @@
 
-# Little Shop of Orders
+# The Pivot
```
Comment (karinamzalez) [inline README.md]: Thank you for doing this!

File: spec/features/visitor_adds_items_to_cart_from_items_pages_spec.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+require 'support/test_helper'
```
Comment (karinamzalez) [inline spec/features/visitor_adds_items_to_cart_from_items_pages_spec.rb]: is this line necessary?

File: spec/features/visitor_adds_items_to_cart_from_items_pages_spec.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+RSpec.feature "Visitor cart additions" do
+  context "Multiple items are available for purchase" do
+    create_test_items
```
Comment (karinamzalez) [inline spec/features/visitor_adds_items_to_cart_from_items_pages_spec.rb]: are you not using factory girl?

**Pull request merged**